app/components/IntroScreen.tsx

```diff
@@ -0,0 +1,52 @@
+'use client';
+
+import { Box } from '@mui/material';
+import bg from '@/public/assets/images/start-screen-image.png';
+import Title from '@/app/components/Title';
+import LanguageModal from '@/app/components/LanguageModal';
+import { useState } from 'react';
+import { locales } from '@/middleware';
+import { AppRouterInstance } from 'next/dist/shared/lib/app-router-context.shared-runtime';
+import { getCookie, setCookie } from 'cookies-next';
+
+interface IntroScreenProps {
+  dict: Dict;
+  router: AppRouterInstance;
+}
+
+const IntroScreen = ({ dict, router }: IntroScreenProps) => {
+  const [isModalOpen, setIsModalOpen] = useState(!getCookie('language'));
+
+  const handleRoute = (localeString: string) => {
+    if (locales.includes(localeString)) {
+      setCookie('language', localeString);
+      router.replace('/' + localeString + '/onboarding');
+    }
+  };
+
+  return (
+    <Box style={{ height: '100%', width: '100%' }}>
+      <Box
+        style={{
```

Comment:
I'm not sure entirely why, but I think you're supposed to be styling MUI-components using the `sx`-prop:
https://mui.com/material-ui/customization/how-to-customize/#1-one-off-customization

Resolve this comment if you don't think it's this is a priority, but please make an issue about looking into this